README.md

```diff
@@ -55,3 +55,6 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 
 If you need any help anywhere in the process, you can always ask a question on our [Gitter Chat](https://gitter.im/jboss-outreach/gci).
 
+#### Signup page design
```

Comment:
I like how you included logo too in the design. Mostly we'll start working on coding designs soon. Thanks for your great efforts! Hope to see more contributions from you we have more design tasks... have a look at them too

Reply:
Also, would you resubmit your task on CGI dashboard. Will approve :)